### PR TITLE
Desktop Edge 2.1.7.0 release notes

### PR DESCRIPTION
**Pull request by usha-mandya**
Release notes for Desktop Edge 2.1.7.0

### CODE REVIEW

File: docker-for-mac/edge-release-notes.md [hunk 1]
```diff
@@ -10,9 +12,29 @@ For information about Stable releases, see the [Stable release
...
 [What to know before you install](install.md#what-to-know-before-you-install).
 
-## Edge Releases of 2019
+## Docker Desktop Community 2.1.7.0
+2019-12-11
 
-### Docker Desktop Community 2.1.6.0
+[Download](https://download.docker.com/mac/edge/41561/Docker.dmg)
+
+> **Note:** Docker Desktop Edge 2.1.7.0 is the release candidate for the upcoming major Stable release. Please help us test this version before the wider release and report any issues in [docker/for-mac](https://github.com/docker/for-mac/issues) GitHub repository.
```
Comment (traci-morrison) [inline docker-for-mac/edge-release-notes.md]: s/issues in/issues in the

File: docker-for-windows/edge-release-notes.md [hunk 2]
```diff
@@ -10,9 +12,39 @@ For information about Stable releases, see the [Stable release
...
 [What to know before you install](install.md#what-to-know-before-you-install).
 
-## Edge Releases of 2019
+## Docker Desktop Community 2.1.7.0
+2019-12-11
 
-### Docker Desktop Community 2.1.6.1
+[Download](https://download.docker.com/win/edge/41561/Docker%20Desktop%20Installer.exe)
+
+> **Note:** Docker Desktop Edge 2.1.7.0 is the release candidate for the upcoming major Stable release. Please help us test this version before the wider release and report any issues in [docker/for-win](https://github.com/docker/for-win/issues) GitHub repository.
```
Comment (traci-morrison) [inline docker-for-windows/edge-release-notes.md]: s/issues in/issues in the

File: docker-for-windows/edge-release-notes.md [hunk 3]
```diff
@@ -10,9 +12,39 @@ For information about Stable releases, see the [Stable release
...
+
+### Upgrades
+
+- [Docker Compose 1.25.1-rc1](https://github.com/docker/compose/releases/tag/1.25.1-rc1)
+
+### New
+
+- Added the **WSL Integration** option in **Settings** > **Resources** to select WSL 2 distributions. This option will be visible only after enabling WSL 2 on your machine.
+- File sharing: The new gRPC FUSE system is no longer experimental and is enabled by default even when experimental features are turned off.
```
Comment (traci-morrison) [inline docker-for-windows/edge-release-notes.md]: Do these acronyms need to be spelled out?

Comment (usha-mandya) [reply docker-for-windows/edge-release-notes.md]: No, I don't think so.

File: docker-for-windows/edge-release-notes.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-windows/edge-release-notes.md]: Fixed.

File: docker-for-mac/edge-release-notes.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-mac/edge-release-notes.md]: Fixed.

**Pull request merged**